### PR TITLE
added a default modal

### PR DESCRIPTION
**Pull request by wendyhub**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,12 +4,22 @@
...
+<<<<<<< Updated upstream
 
     <!-- Bootstrap, jQuery -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
     <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
 
     <!-- Main CSS and JS -->
+=======
+    
+    //
```
Comment (Winyumi) [inline index.html]: Is this supposed to be here? Double slash comments do not work in HTML.
```suggestion
```

**Review by jondam1985 — APPROVED** [review]
Approved

**Review by wendyhub — COMMENTED** [review]
test

**Review by Winyumi — CHANGES_REQUESTED** [review]
Here's some feedback.

**Review by Winyumi — APPROVED** [review]

**Pull request merged**